.github/workflows/build.yml

```
@@ -98,6 +98,7 @@ jobs:
         env:
           PLATFORM_ARTIFACT_MARKER: ${{ matrix.plan.platform.artifactMarker }}
           ARTIFACT_SELECTOR: ${{ matrix.plan.mode.artifactSelector }}
+          FEATURES_MARKER: ${{ matrix.plan.mode.artifactFeaturesMarker }}
```

```suggestion
          ARTIFACT_MARKER: ${{ matrix.plan.mode.artifactMarker }}
```
Better use the same naming everywhere

---

Certainly, that will be more systematic